src/GlobalSearch/DateOfBirth.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import DateField from '@govuk-react/date-field'
+import moment from 'moment'
+
+class DateOfBirth extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: {},
+      dateIsValid: false,
+      dateIsInThePast: true,
+      blank: true,
+    }
+    this.onChange = this.onChange.bind(this)
+    this.errorText = this.errorText.bind(this)
+  }
+
+  componentDidMount() {
+    this.onChange({})
+  }
+
+  onChange(value) {
```

Comment:
```suggestion
  onChange = value => {
```

Reply:
Not a deal breaker but using class properties like ^ means you don't have to bind it in the constructor.

Reply:
Done